compiler/il/ILOpCodeProperties.hpp

```diff
@@ -3474,11 +3474,11 @@
    /* .opcode               = */ TR::acmpeq,
    /* .name                 = */ "acmpeq",
    /* .properties1          = */ ILProp1::BooleanCompare,
-   /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE,
+   /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::UnsignedCompare | ILProp2::SupportedForPRE,
```

Comment:
I'm not sure whether we want to say that this is an unsigned comparison, when (in)equality is insensitive to signedness